randovania/games/am2r/logic_database/The Tower.txt

```diff
@@ -114,7 +114,7 @@ Extra - liquid_info: {'liquid_type': 0, 'liquid_level': 0}
   > Door to Plasma Beam Chamber Access
       Any of the following:
           Space Jump Wall
-          Hi-Jump Boots and Mid-Air Morph (Intermediate)
+          Hi-Jump Boots and Morph Ball and Mid-Air Morph (Intermediate)
```

Comment:
I'm unsure right now if this is supposed to be a morph glide instead.

Reply:
glide
![image](https://github.com/user-attachments/assets/47524f3a-a906-423c-909c-9799426a70fd)